.circleci/config.yml

```diff
@@ -91,352 +91,6 @@ parameters:
     default: ""
 
 commands:
-  unittest_tox:
```

Comment:
Previously I only removed job names from the section at the bottom where workflows are defined (aka which jobs run for which workflow), but removing it all together from this file is fine as well (if we ever need to add it back for some reason, there is always git history :)).